src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -154,14 +157,14 @@ public void cleanEmpty() {
                             @ApiResponse(code = 401, message = "User not authenticated"),
                             @ApiResponse(code = 403, message = "Permission denied"), })
     @RequestMapping(value = "/{bucketName}", method = DELETE)
-    @ResponseStatus(HttpStatus.FORBIDDEN)
-    public ResponseEntity<?> delete(
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Are you sure you want to change this FORBIDDEN to OK?

Reply:
Yes this forbidden was an error that hidden by implementation. Now that the implementation is fixed the correct value has to be set.